web/src/pages/starter-kits/general.tsx

```diff
@@ -0,0 +1,158 @@
+import { NavBar } from "@/components/navbar/NavBar";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { Card } from "@/components/starter-kits/Card";
+import { StepInfo } from "@/components/starter-kits/StepInfo";
+import { getMergedConfig } from "@/contexts/configContext";
+import { ROUTES } from "@/lib/domain-logic/routes";
+import { buildUserRoadmap } from "@/lib/roadmap/buildUserRoadmap";
+import { Roadmap, Task } from "@/lib/types/types";
+import analytics from "@/lib/utils/analytics";
+import { ProfileData, createEmptyProfileData } from "@businessnjgovnavigator/shared/index";
+import type { InferGetStaticPropsType } from "next";
+import { useRouter } from "next/router";
+import { ReactElement } from "react";
+
+const General = (props: InferGetStaticPropsType<typeof getStaticProps>): ReactElement => {
+  const router = useRouter();
+  const Config = getMergedConfig();
+
+  const getTaskNamesForStep = (tasks: Task[], step: number): string[] => {
+    return tasks.filter((task) => task.stepNumber === step).map((task) => task.name);
+  };
+
+  return (
+    <PageSkeleton landingPage={true}>
+      <NavBar isSeoStarterKit={true} />
+      <main className="desktop:grid-container-widescreen desktop:padding-x-7">
+        <section
+          aria-label="Introduction"
+          className="bg-cool-extra-light window-width-shadow-cool-extra-light flex flex-justify padding-y-4 desktop:padding-y-8 padding-x-2 desktop:padding-x-0"
+        >
+          <div className={"text-center desktop:text-left desktop:grid-col-5"}>
+            <Heading level={1} styleVariant="h1Large" className="display-only-desktop">
+              {Config.starterKits.hero.title}
+              <br />
+            </Heading>
+            <Heading level={1} className="display-none-desktop">
+              {Config.starterKits.hero.title}
+              <br />
+            </Heading>
+            <div className="text-base-darkest font-sans-md margin-bottom-3">
+              {Config.starterKits.hero.subtitle}
+            </div>
+            <PrimaryButton
+              isColor={"secondary-vivid-dark"}
+              onClick={() => {
+                analytics.event.starter_kit_landing.click.start_now_button();
+                router.push(ROUTES.onboarding);
+              }}
+              isRightMarginRemoved={true}
+              isLargeButton={true}
+            >
+              {Config.starterKits.hero.ctaButton}
+            </PrimaryButton>
+          </div>
+
+          <div className="desktop:grid-col-6 desktop:display-block display-none">
+            <img className="width-100" src="/img/seo-img-main.webp" alt="" role="presentation" />
+          </div>
+        </section>
+        <section
+          aria-label="Solutions"
```

Comment:
[dust] Does a Section need an `aria-label` when it clearly has a heading already? Also, we try not to encode bare strings in our code like this.

Reply:
For acessibility navigation purposes it does but to be honest I'm not entirely sure a section tag is necessary at all

Reply:
I guess for semantics during dev it's helpful. The labels aren't really meaningful as is to screen readers IMO.